### PR TITLE
Add only to Scenario type definition

### PR DESCRIPTION
**Pull request by Akxe**

### CODE REVIEW

File: lib/command/definitions.js [hunk 1]
```diff
@@ -38,6 +40,46 @@ interface ILocator {
...
+  _beforeStep(step: () => void): void
+  /** Hook executed after each step */
+  _afterStep(step: () => void): void
+  /** Hook executed before each suite */
+  _beforeSuite(suite: () => void): void
+  /** Hook executed after each suite */
+  _afterSuite(suite: () => void): void
+  /** Hook executed after all tests are executed */
+  _finishTest(suite: () => void): void
+  /**Access another configured helper: `this.helpers['AnotherHelper']` */
```
[the remark this answers is not in the document]
Comment (Akxe) [reply lib/command/definitions.js]: It is a comment... it's okay

**Comment by Akxe** [comment]
I have no idea why travis fails...

**Comment by DavertMik** [comment]
Not sure either, but something terrible is happening there. 
No output is printed in all `run` commands...

**Comment by Akxe** [comment]
I have only edited type declaration... It should not have been me, right?

**Comment by DavertMik** [comment]
Ok. Let's try to merge it and see how it goes.

**Comment by DavertMik** [comment]
Well, it was you :) 

In this line:

```
`this.helpers['AnotherHelper']`
```

**Comment by Akxe** [comment]
@DavertMik But that is a comment... :D It should not affect code... But thanks :)

**Pull request merged**